### ISSUE TITLE
can't build lagoon/keycloak because of missing yum command in container

### ISSUE THREAD

**Issue by renebakx** [uselagoon/lagoon#1420]
**Describe the bug**
Make build stops at building the keycloak image.

```make build
docker build --quiet --build-arg LAGOON_VERSION=v1.1.4 --build-arg IMAGE_REPO=lagoon -t lagoon/keycloak -f services/keycloak/Dockerfile services/keycloak
Sending build context to Docker daemon  82.94kB
Step 1/23 : ARG IMAGE_REPO
Step 2/23 : FROM ${IMAGE_REPO:-lagoon}/commons as commons
 ---> 2b3153a22f87
Step 3/23 : FROM jboss/keycloak
 ---> b642d96ad2e5
Step 4/23 : USER root
 ---> Using cache
 ---> c9c1c9393fa0
Step 5/23 : RUN yum install -y nc     && yum clean all     && rm -rf /var/cache/yum
 ---> Running in 67d21a74a548
/bin/sh: yum: command not found
The command '/bin/sh -c yum install -y nc     && yum clean all     && rm -rf /var/cache/yum' returned a non-zero code: 127
make: *** [build/keycloak] Error 127

```

**To Reproduce**
1. Checkout amazeeio/lagoon on tag 1.1.4 (on macos 10.13 but that not really relevant)
2. run make build
3. wait a long time until the process starts to build lagoon/keycloak
4. the error mentioned above appears.

**Expected behavior**
That the build process continues ;)

**Additional context**
Docker engine 19.03.4, compose 1.25.0-rc on Docker Desktop 2.1.5.0 (edge) running on MacOS 10.13

The base image lagoon/commons is based on Alpine, alpine has no yum command. Not sure if the upstream image from RedHat for jboss/keycloak has yum. I would assume it has, given that it's a redhat image.

Removing the yum chain from the docker file solves the issue, but i'm not sure that it actually solves it. (aka I have no clue what i'm doing there ;))

**Comment by rocketeerbkw** [comment]
@renebakx are you running the latest lagoon? this (and some other keycloak dockerfile changes) was fixed on master already: https://github.com/amazeeio/lagoon/commit/6baa98f6d0ba903bddaad18351bde9f0995fe377

**Comment by renebakx** [comment]
> @renebakx are you running the latest lagoon? this (and some other keycloak dockerfile changes) was fixed on master already: [6baa98f](https://github.com/amazeeio/lagoon/commit/6baa98f6d0ba903bddaad18351bde9f0995fe377)


nope, not master. Wanted to test it with a tagged release. Policy forbids me to run not tagged code in production. But given it is a sandbox / testing env. I can get away with it for now ;)

Thanks! Closing this issue :)
